pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/ArrayBaseTransformFunctionTest.java

```diff
@@ -76,6 +78,101 @@ public void testArrayTransformFunction() {
     }
   }
 
+  @Test
+  public void testArrayNullColumn() {
+    ExpressionContext expression =
+        RequestContextUtils.getExpression(String.format("%s(%s)", getFunctionName(), INT_MV_NULL_COLUMN));
+    TransformFunction transformFunction = TransformFunctionFactory.get(expression, _dataSourceMap);
+    Assert.assertEquals(transformFunction.getClass().getName(), getArrayFunctionClass().getName());
+    Assert.assertEquals(transformFunction.getName(), getFunctionName());
+    Assert.assertEquals(transformFunction.getResultMetadata().getDataType(), getResultDataType(FieldSpec.DataType.INT));
+    Assert.assertTrue(transformFunction.getResultMetadata().isSingleValue());
+    Assert.assertFalse(transformFunction.getResultMetadata().hasDictionary());
+
+    switch (getResultDataType(FieldSpec.DataType.INT)) {
+      case INT:
+        Pair<int[], RoaringBitmap> intResults = transformFunction.transformToIntValuesSVWithNull(_projectionBlock);
```

Comment:
Why is `SVWithNull` being called here? Shouldn't it be`MVWithNull`

Reply:
I think that `SV` here represents the output. For instance, the output of `ArrayAvg` is single value.